src/App.tsx

```diff
@@ -174,12 +144,11 @@ const App = () => {
           <Button
             variant="light"
             onClick={() => {
-              if (!formRef.current) return
               const urlWithAppData = getURLwithAppData({
                 schema,
-                values: validations
-                  .map(({value}) => value)
-                  .filter((value): value is string => typeof value == 'string'),
+                values: values.filter(
+                  (value): value is string => typeof value == 'string',
```

Comment:
What's the difference between

```
(value): value is string => typeof value == 'string'
```

and

```
(value): boolean => typeof value == 'string'
```

Reply:
The first one is a type predicate function. After you call the function, the value you passed in will have a recognized type. In this specific case, it doesn't bring any added value. I can remove it